test/e2e/tests/tokens/defi/view-defi-details.spec.ts

```diff
@@ -32,21 +37,37 @@ describe('View DeFi details', function () {
 
         const defiTab = new DeFiTab(driver);
 
-        // check ethereum positions present
-        await switchToNetworkFlow(driver, 'Ethereum Mainnet');
+        // check ethereum positions present)
+        if (!isGlobalNetworkSelectorRemoved) {
+          await switchToNetworkFlow(driver, 'Ethereum Mainnet');
+        }
+
         await defiTab.check_groupIconIsDisplayed();
         await defiTab.defiTabCells.check_tokenName('Aave V3');
         await defiTab.defiTabCells.check_tokenMarketValue('$14.74');
         await defiTab.defiTabCells.check_tokenName('Aave V2');
         await defiTab.defiTabCells.check_tokenMarketValue('$0.33');
 
-        // switch popular networks and check linea positions present
-        await defiTab.openNetworksFilterAndClickPopularNetworks();
+        if (!isGlobalNetworkSelectorRemoved) {
+          await defiTab.openNetworksFilterAndClickPopularNetworks();
+        }
         await defiTab.defiTabCells.check_tokenName('UniswapV3');
         await defiTab.defiTabCells.check_tokenMarketValue('$8.48');
         await defiTab.defiTabCells.check_tokenName('UniswapV2');
         await defiTab.defiTabCells.check_tokenMarketValue('$4.24');
 
+        // deselect linea
+        // this feels wrong, there might be a potential bug here with defi
```

Comment:
Could you expand on this? As in we need to re-sort our lists to find "Linea Mainnet"?